cypress/plugins/dateTools.js

```diff
@@ -0,0 +1,6 @@
+
+export function getCurrentDate(){
```

Comment:
it's not plugin, just util

Reply:
we don't have such folder. Do you mean I should add new folder utils in the root?

Reply:
moved to utils in support